drivers/virtualbox/virtualbox.go

```diff
@@ -229,7 +232,11 @@ func (d *Driver) Create() error {
 	}
 
 	if d.IsVTXDisabled() {
-		return ErrMustEnableVTX
+		// Let's log a warning to warn the user. When the vm is started, logs
+		// will be checked for an error anyway.
+		// We could fail right here but the method to check didn't prove being
+		// bulletproof.
+		log.Warn("This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory.")
```

Comment:
I agree that this being hidden away in the plugin output (behind `--debug`) is not ideal -- in the future we can talk about how to improve the situation.  Let me know if you have any ideas.